platform/android/scripts/metrics.sh

```diff
@@ -12,6 +12,7 @@ scripts/check_binary_size.js "platform/android/MapboxGLAndroidSDK/build/intermed
 # Track overall library size
 scripts/check_binary_size.js "platform/android/MapboxGLAndroidSDK/build/outputs/aar/MapboxGLAndroidSDK-release.aar" "Android AAR"
 
-if [[ $CIRCLE_BRANCH == master ]]; then
-    scripts/publish_binary_size.js
-fi
+# Build source data for http://mapbox.github.io/mapbox-gl-native/metrics/binary-size/
+scripts/publish_binary_size.js
```

Comment:
Is it intended to publish metrics for every commit on every branch? I think that would be very noisy and make and graphs hard to read. Like we did before, it would make more sense to log only for commits to master, so that long term we can track these changes. For PRs and other commits, the Github Checks already provide this information for consideration during review.